cfpb/cfgov-refresh
# Update cfpb-chart-builder to 6.2.0

Adds max range to bar chart navigator of 7 years.

## Changes

- Update cfpb-chart-builder to `6.2.0` from `6.1.3`

## Testing

1. Pull branch, `./frontend.sh`, and run a local server.
2. Visit http://localhost:8000/data-research/consumer-credit-trends/credit-cards/lending-borrower-age/ and scroll down to the bar charts and see that the navigator range handles can't be dragged to more than a 7 year range.

## Screenshots

<img width="699" alt="Screen Shot 2019-10-02 at 12 17 28 PM" src="https://user-images.githubusercontent.com/704760/66062037-9f835d00-e50e-11e9-9b9f-dcd8a10d977a.png">
